Formula/grafana-agent.rb

```diff
@@ -61,15 +61,13 @@ def caveats
     (testpath/"grafana-agent.yaml").write <<~EOS
       server:
         log_level: info
-        http_listen_port: #{port}
-        grpc_listen_port: #{free_port}
     EOS
 
     system "#{bin}/grafana-agentctl", "config-check", "#{testpath}/grafana-agent.yaml"
 
     fork do
       exec bin/"grafana-agent", "-config.file=#{testpath}/grafana-agent.yaml",
-        "-metrics.wal-directory=#{testpath}/wal"
+        "-metrics.wal-directory=#{testpath}/wal", "-server.http.address=127.0.0.1:#{port}"
```

Comment:
Shouldn't this also set grpc? `--server.grpc.address=127.0.0.1:#{free_port}`

Reply:
it isn't needed for the test -- the test only curls the http port /metrics endpoint